File: Backend_datospersonales/app/src/main/java/com/oscarmena/backend_datospersonales/UsuarioResponse.java
package com.oscarmena.backend_datospersonales;

public class UsuarioResponse {
    private String identificacion;
    private String documento;
    private String nombres;
    private String apellidos;
    private String fechaNacimiento;
    private String email;
    private String direccion;
    private String imagen;

    // Getters
    public String getIdentificacion() { return identificacion; }
    public String getDocumento() {
        return documento; }
    public String getNombres() { return nombres; }
    public String getApellidos() { return apellidos; }
    public String getFechaNacimiento() { return fechaNacimiento; }
    public String getEmail() { return email; }
    public String getDireccion() { return direccion; }
    public String getImagen() { return imagen; }
}

File: Backend_datospersonaleskotlin/app/src/main/java/com/oscarmena/backend_datospersonaleskotlin/UsuarioResponse.kt
package com.oscarmena.backend_datospersonaleskotlin

data class UsuarioResponse(
    val identificacion: String,
    val documento: String,
    val nombres: String,
    val apellidos: String,
    val fechaNacimiento: String,
    val email: String,
    val direccion: String,
    val imagen: String
)